# Repeatome and genome size evolution in Scyphozoa jellyfishes (Medusozoa: Cnidaria)
**Poster in Biodiversity Genomics 2020.**
Presenter: Santander Mylena D. mylena.santander@usp.br  
Santander MD., Maronna MM, Gamero-Mora E., Molinari CG., Lawley JW., Stampar SN., Morandini AC., Andrade SC.


## References
Poster background image: 
**Plate 35: Stomolophus. Haeckel E. 1979. Das System der Medusen. Gustav Fisher, Jena.**

1. Kim, Y. B., Oh, J. H., McIver, L. J., Rashkovetsky, E., Michalak, K., Garner, H. R., ... & Michalak, P. (2014). Divergence of *Drosophila melanogaster* repeatomes in response to a sharp microclimate contrast in Evolution Canyon, Israel. Proceedings of the National Academy of Sciences, 111(29), 10630-10635.
2. Bosco, G., Campbell, P., Leiva-Neto, J. T., & Markow, T. A. (2007). Analysis of Drosophila species genome size and satellite DNA content reveals significant differences among strains as well as between species. Genetics, 177(3), 1277-1290.
3. Elliott, T. A., & Gregory, T. R. (2015). What’s in a genome? The C-value enigma and the evolution of eukaryotic genome content. Philosophical Transactions of the Royal Society B: Biological Sciences,  370, 20140331
4. Naville, M., Henriet, S., Warren, I., Sumic, S., Reeve, M., Volff, J.-N., & Chourrout, D. (2019). Massive changes of genome size driven by expansions of non-autonomous transposable elements. Current Biology, 29(7), 1161-1168.e6. 
5. Gregory, T. R. (2001). Coincidence, coevolution, or causation? DNA content, cell size, and the C-value enigma. Biological Reviews, 76(1), 65–101. 
6. Adachi, K., Miyake, H., Kuramochi, T., Mizusawa, K., & Okumura, S. I. (2017). Genome size distribution in phylum Cnidaria. Fisheries science, 83(1), 107-112.
7. WoRMS Editorial Board (2019). World Register of Marine Species. Available at http://www.marinespecies.org. [Last accesed in 29/09/2020]. 
8. DeSalle, R., Giribet, G., & Wheeler, W. (2002). Techniques in molecular systematics and evolution. Springer Science & Business Media.
9. Goubert, C., Modolo, L., Vieira, C., ValienteMoro, C., Mavingui, P., & Boulesteix, M. (2015). De novo assembly and annotation of the Asian tiger mosquito (Aedes albopictus) repeatome with dnaPipeTE from raw genomic reads and comparative analysis with the yellow fever mosquito (*Aedes aegypti*). Genome biology and evolution, 7(4), 1192-1205.
10. Gregory, T.R. (2020). Animal Genome Size Database. Available at http://www.genomesize.com.  [Last accesed in 29/09/2020]
11. Nong, W., Cao, J., Li, Y., Qu, Z., Sun, J., Swale, T., ... & Bendena, W. (2020). Jellyfish genomes reveal distinct homeobox gene clusters and conservation of small RNA processing. Nature communications, 11(1), 1-11.
12. Li, Y., Gao, L., Pan, Y., Tian, M., Li, Y., He, C., ... & Zhou, Z. (2020). Chromosome-level reference genome of the jellyfish *Rhopilema esculentum*. GigaScience, 9(4), giaa036.
13. Dolezel, J., & Bartoš, J. (2005). Plant DNA flow cytometry and estimation of nuclear DNA content. Ann Bot, 95(1), 99-110.

## Software used
* [Seqyclean v1.10.9](https://github.com/ibest/seqyclean)
* [dnaPipeTE v1.3](https://github.com/clemgoub/dnaPipeTE)
* [BBsplit from BBMap](https://sourceforge.net/projects/bbmap/)
* [NOVOPlasty v4.2](https://github.com/ndierckx/NOVOPlasty)
* Python3 libraries: [ETE3 v3.1.2](http://etetoolkit.org/), [Scipy v1.5.2](https://www.scipy.org/), [Matplotlib 3.3.1](https://matplotlib.org/) 
